nx.json

```diff
@@ -0,0 +1,81 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "lint",
+          "typecheck",
+          "test-unit",
+          "test-build",
+          "test-docs",
+          "docs-build"
+        ],
+        "accessToken": "NjYwZGFlNWYtZGYyNS00MTIzLTkzYTEtYzAxZTZiYzUwYTVifHJlYWQtd3JpdGU="
+      }
+    }
+  },
+  "namedInputs": {
+    "default": ["{workspaceRoot}/packages/**/*"],
+    "production": [
+      "default",
+      "!{workspaceRoot}/packages/**/?(*.)+(spec|test).[jt]s?(x)?(.snap)",
+      "!{workspaceRoot}/vitest.config.e2e.ts",
+      "!{workspaceRoot}/vitest.config.ts",
+      "!{workspaceRoot}/.eslintignore",
+      "!{workspaceRoot}/.eslintrc.cjs",
+      "!{projectRoot}/tsconfig.check.json"
+    ],
+    "playground": [
+      "default",
+      "{workspaceRoot}/playground/**/*",
+      {
```

Comment:
`inputs` can include environment variables or the result of a shell script.

Reply:
would it also include extra cli arguments? eg `pnpm test-serve  some-test-file-to-focus.ts`

Reply:
yeah, extra args should be included already